docs/pact-reference.md

```diff
@@ -1104,33 +1105,31 @@ Tables and objects can only take a schema type literal.
 Special forms {#special-forms}
 ---
 
-### Docs and metadata
+### Docs and Metadata
 Many special forms like [defun](#defun) accept optional documentation strings,
 in the following form:
 
 ```lisp
-(defun foo (bar)
-  "Do the thing with BAR"
-  ...)
+(defun average (a b)
+  "take the average of a and b"
+  (/ (+ a b) 2))
 ```
 
-However, in this position an optional _metadata section_ can specify docs and
-metadata, where metadata can be tagged with any key desired.
-The following code provides a docstring of "does the thing with BAR" and specifies metadata of type
-`property` and `example`:
+However, in this position, extra metadata fields can also be specified.
+One such metadata field is `@model`, which represents a *property* that can
+be used by Pact tooling to verify the correctness of the implementation:
 
 ```lisp
-(defun foo (bar)
-  ("does the thing with BAR"
-    (property [(when something abort)])
-    (example (foo "my house")))
-  ...)
+(defun average (a b)
+  @doc   "take the average of a and b"
+  @model (property (= (+ a b) (* 2 result)))
+  (/ (+ a b) 2))
 ```
 
-Thus, the metadata form is DOC PAIR*, where a PAIR is (ATOM EXPR). The Pact
-language lexer/compiler ignores all EXPR forms, to be lexed/compiled at some later stage
-by whatever tool recognizes ATOM.
+Indeed, a bare docstring like `"foo"` is actually just a short form for `@doc "foo"`.
+More metadata types are expected to be added in future.
```

Comment:
Are they?

Reply:
I could've sworn that this had been implied in the original text, or that somebody had told me this directly. Now I can't find evidence, though.

Reply:
I think there's no reason to say this, even if we did in the past, as it doesn't add information, and isn't an actual plan -- let's remove it.